<launch>
    <node name = "rrp_ik_server_node" 
          pkg = "rbe500_project" 
          type = "rrp_ik_server.py" 
          output = "screen"/>

    <node name = "rrp_pid_node"
          pkg = "rbe500_project"
          type = "rrp_pid.py"
          output = "screen"/>

</launch>
